src/components/Common/MultipleComboBox.tsx

```diff
@@ -0,0 +1,296 @@
+                      index,
+                    })}
+                  >
+                    {selectedItemRow ? (
+                      createElement(selectedItemRow, {
+                        item: selectedItemForRender,
+                      })
+                    ) : (
+                      <span className="max-w-[150px] truncate break-all text-xs">
+                        {getItemLabel(selectedItemForRender)}
+                      </span>
+                    )}
+                    <span
+                      data-qa={`unselect-item-${getItemValue(
+                        selectedItemForRender,
+                      )}`}
+                      className="cursor-pointer px-1"
+                      onClick={(e) => {
+                        e.stopPropagation();
+                        removeSelectedItem(selectedItemForRender);
+                      }}
+                    >
+                      <IconX height={12} width={12} />
+                    </span>
+                  </span>
+                );
+              })}
+
+            <input
+              disabled={disabled}
+              placeholder={placeholder || ''}
+              className="bg-transparent px-3 py-1 outline-none placeholder:text-secondary"
+              {...getInputProps(
+                getDropdownProps({
+                  preventKeyAction: isOpen,
+                  ref: refs.reference as RefObject<HTMLInputElement>,
+                }),
+              )}
+            />
+          </div>
+          <ul
+            className={classNames(
+              'z-10 max-h-80 overflow-auto rounded bg-layer-0',
+              !isOpen && 'hidden',
+            )}
+            {...getMenuProps(
+              { ref: refs.floating as RefObject<HTMLUListElement> },
+              { suppressRefError: true },
+            )}
+            style={{
+              position: strategy,
+              top: y ?? '',
+              left: x ?? '',
+              width: `${floatingWidth}px`,
+            }}
+          >
+            {isOpen &&
+              (displayedItems?.length > 0 ? (
+                displayedItems.map((item, index) => (
+                  <li
+                    className={classNames(
+                      'group flex h-[34px] cursor-pointer flex-col justify-center px-3',
+                      highlightedIndex === index && 'bg-accent-primary-alpha',
+                      selectedItem === item && 'bg-accent-primary-alpha',
+                    )}
+                    key={`${getItemValue(item)}${index}`}
+                    {...getItemProps({ item, index })}
+                  >
+                    {itemRow
+                      ? createElement(itemRow, { item })
+                      : getItemLabel(item)}
+                  </li>
+                ))
+              ) : (
+                <li className="px-3 py-2">
+                  {notFoundPlaceholder || t('No available items')}
+                </li>
```

Comment:
I don't think we need to show it
![image](https://github.com/epam/ai-dial-chat/assets/20163971/91705a8e-d09e-45de-a131-7b03929a57dd)
I suggest to not show it if input is empty:
```suggestion
              ) : (inputValue &&
                <li className="px-3 py-2">
                  {notFoundPlaceholder || t('No available items')}
                </li>
```

Reply:
Changed as suggested.